examples/pyLiveView.py

```diff
@@ -144,14 +144,20 @@ def run(self):
       lst = SonyAPI.LiveviewStreamThread(url)
       lst.start()
 
+      versions = camera.getVersions()
+      if versions and type(versions) == dict and 'result' in versions.keys():
```

Comment:
Very minor suggestion, but idiomatic Python discourages explicit use of type checking like this, in favor of so-called ["duck typing"](https://realpython.com/lessons/duck-typing/).

This line could be simplified to:

`if versions and 'result' in versions:`